build.gradle

```diff
@@ -85,9 +84,9 @@ dependencies {
 	}
 
 	// tinyfile management
-	implementation ('net.fabricmc:tiny-remapper:0.8.7')
+	implementation ('net.fabricmc:tiny-remapper:0.8.8')
 	implementation 'net.fabricmc:access-widener:2.1.0'
-	implementation 'net.fabricmc:mapping-io:0.2.1'
+	implementation 'net.fabricmc:mapping-io:0.4.2'
 
 	implementation ('net.fabricmc:lorenz-tiny:4.0.2') {
```

Comment:
Might be a good idea to build a new copy of lorenz-tiny against the latest mapping-io.